docs/quickstarts/nextjs.mdx

```diff
@@ -118,55 +125,88 @@ The [`<ClerkProvider>`](/docs/components/clerk-provider) component provides acti
   </Tab>
 </Tabs>
 
-### Require authentication to access your app
+### Add authentication to your app
 
-Now that Clerk is installed and mounted in your application, you can decide which pages are public and which should require authentication to access.
+Now that Clerk is installed and mounted in your application, you can decide which pages are public and which should require authentication to access:
 
-Create a `middleware.ts` file. If your application uses the `src/` directory, your `middleware.ts` file should be placed inside the `src/` folder. If you are not using a `src/` directory, place the `middleware.ts` file in your root directory alongside `.env.local`.
+1. Create a `middleware.ts` file at the root of your project, or in your `src/` directory if you have one.
+2. In your `middleware.ts`, export Clerk's [`authMiddleware`](https://clerk.com/docs/references/nextjs/auth-middleware) helper:
+    ```tsx filename="middleware.ts" {1,3}
+    import { authMiddleware } from "@clerk/nextjs";
 
-Copy and paste the following code into your `middleware.ts`. With this, your entire application is protected.
+    export default authMiddleware({});
+    ```
+3. With `authMiddleware()`, authentication will be enabled by default on all routes that your Next.js middleware runs on. You can specify valid routes using Next.js's [`matcher`](https://nextjs.org/docs/app/building-your-application/routing/middleware#matcher). Add the following code to your `middleware.ts` to protect your entire application:

```

Comment:
I have another comment below, but with `authMiddleware()` we suggest matching all routes as its built with an opt-out approach. The key here is that routes are auth enabled and protected.

From there, the developer would use one of two things to make routes accessible:
1. `publicRoutes: []` - this leaves the route with auth information available, but means that users who are not signed in can access the route.
2. `ignoredRoutes: []` - there is no protection and no auth information. Removing a route from the matcher would be equal to an `ignoredRoute: []`.

Reply:
Thanks, I've updated this paragraph to reflect this now, and also added `publicRoutes` and `ignoredRoutes` to the code snippet to introduce the concept to readers